docs/source/science/ATL08/ATL08.ipynb

```diff
@@ -0,0 +1,1386 @@
+{
```

Comment:
<p>Link to the Pangeo workspace definition? and maybe the getting started for how to pick a workspace?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/362/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Should mention, the <code>Pangeo</code> workspace is recommended for this tutorial. </p>